File: 1/02.rb
# -*- coding: utf-8 -*-

=begin
「パトカー」＋「タクシー」の文字を先頭から交互に連結して文字列「パタトクカシーー」を得よ．
=end

str1 = 'パトカー'
str2 = 'タクシー'

i = 0
while (c1 = str1[i]) &&  (c2 = str2[i])
  print "#{c1}#{c2}"
  i += 1
end

File: python/1/02.py
# -*- coding: utf-8 -*-

'''
「パトカー」＋「タクシー」の文字を
先頭から交互に連結して
文字列「パタトクカシーー」を得よ．
'''

str1 = u'パトカー'
str2 = u'タクシー'
str3 = u''

for i,j in zip(str1, str2):
  str3 += i+j

print str3
